### PR TITLE
Updated with deployment circuit breaker

### PR DESCRIPTION
**Pull request by flywheelnz**
## what
* Updated to support deployment circuit breaker and optional rollback

## why
* AWS feature compatability

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -348,3 +348,15 @@ variable "exec_enabled" {
   description = "Specifies whether to enable Amazon ECS Exec for the tasks within the service"
   default     = false
 }
+
+variable "deployment_circuit_breaker" {
```
Comment (joe-niland) [inline variables.tf:352]: @flywheelnz to go with the general naming convention I'd suggest `circuit_breaker_deployment_enabled`

File: variables.tf [hunk 2]
```diff
@@ -348,3 +348,15 @@ variable "exec_enabled" {
...
+  type        = bool
+  description = "Whether to enable the deployment circuit breaker logic for the service"
+  default     = false
+}
+
+variable "circuit_breaker_enable_rollback" {
```
Comment (joe-niland) [inline variables.tf:358]: I'd suggest `circuit_breaker_rollback_enabled`

**Review by joe-niland — DISMISSED** [review]
Thanks @flywheelnz, just some minor things

**Comment by joe-niland** [comment]
@flywheelnz please allow maintainer edits on your PR so the automated formatting, etc can run

File: variables.tf (hunk 1 above)
Comment (flywheelnz) [reply variables.tf]: Done!

**Comment by flywheelnz** [comment]
@joe-niland Not sure we can add maintainer edits to an organisation repository, so I've added cloudpossebot manually - let me know if there's any issues.

**Comment by joe-niland** [comment]
@flywheelnz [you're right](https://github.com/isaacs/github/issues/1681). unfortunately this didn't allow access to Github Actions. Possible to resubmit from an individual account?

**Comment by flywheelnz** [comment]
@joe-niland Fixed!

**Comment by joe-niland** [comment]
@flywheelnz thanks - getting somewhere now! I think maybe a find-replace renamed the block name. Please have a look. Also if you can try running `make pr/auto-format` locally it may resolve some other issues.

**Comment by flywheelnz** [comment]
Sorry for the delay on this - we're sure busy at the moment.
Fixed the accidental block replacement. Ran `make pr/auto-format`

**Review by joe-niland — APPROVED** [review]

**Comment by joe-niland** [comment]
Thanks @flywheelnz busy is good! :)
This has been released. Just FYI, some upcoming module refactoring (see release 0.57.0) will possibly remove release 0.58.0, so you may need to adjust the version in your TF code. Will try to notify you.

**Pull request merged**